### ISSUE TITLE
RapidClient

### ISSUE THREAD

**Issue by ivangerasimov39** [veyesys/version#4]
Is it possible at the system start to open the RapidClient with two tabs (View1 and View2) or to open saved views?
If not, it would be cool to implement this
Example saved views:
![image](https://user-images.githubusercontent.com/29921722/28207997-ee8f5f68-688b-11e7-9698-9d36df8d99b1.png)

**Comment by xsmart** [comment]
Yes, we can add , but maybe this is not a common requirement.you can add by yourself.

**Comment by ivangerasimov39** [comment]
How to add?
